### PR TITLE
Store anonymous ID in localStorage

### PR DESCRIPTION
**Pull request by nhi-nguyen**

### CODE REVIEW

**Comment by nhi-nguyen** [comment]
TBD: tests

**Comment by f2prateek** [comment]
LGTM! Just needs tests.

File: lib/user.js [hunk 1]
```diff
@@ -99,12 +100,25 @@ User.prototype.anonymousId = function(anonymousId) {
...
     store.set('ajs_anonymous_id', anonymousId);
+    localStorage.set('ajs_anonymous_id', anonymousId);
     return this;
   }
 
   // new
   anonymousId = store.get('ajs_anonymous_id');
   if (anonymousId) {
+    // value exist in cookie, copy it to localStorage
+    localStorage.set('ajs_anonymous_id', anonymousId);
```
Comment (nhi-nguyen) [inline lib/user.js]: @f2prateek this line is new (was not in the original code you reviewed.) Please have another look!

Comment (f2prateek) [reply lib/user.js]: Yep I saw this -  this makes sense to me!

**Pull request merged**